src/renderer/styles/about-app.less

```diff
@@ -121,7 +121,30 @@ body {
     font-size: 0.8rem;
     text-align: center;
     padding: 10px 32px;
-    margin: 4px 20px;
+    margin: 4px 8px;
+    display: inline-block;
+    text-decoration: none;
+    line-height: 12px;
+    background-color: rgba(61, 162, 253, 1);
```

Comment:
You can define a variable for this color

Reply:
Im going to change these into a variable that we can possibly adapt with the Design System of Symphony
I.E: [Electricity05] -> 
![image](https://user-images.githubusercontent.com/97150869/193747812-42735f13-2622-4bd4-b4bb-5b42070d1b85.png)

Reply:
Done, adding new color palette section